app/src/features/surveys/CreateSurveyPage.tsx

```diff
@@ -377,6 +383,47 @@ const CreateSurveyPage = () => {
 
                 <Divider className={classes.sectionDivider} />
 
+                <HorizontalSplitFormComponent
+                  title="Site Selection Strategy"
+                  summary="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed."
+                  component={
+                    <>
```

Comment:
For now (pending some upcoming Survey from changes), I think we should leave this markup inline, since this code just handles showing and hiding the particular Survey form elements.

Reply:
Actually, I'll break this out into a Sample Method component

Reply:
yeah it just keeps this page a little cleaner and easier to manage 🧹